test/protocol/OrchestrationHandlerTest.js

```diff
@@ -253,13 +253,13 @@ describe("IBosonOrchestrationHandler", function () {
         adminDR.address,
         clerkDR.address,
         treasuryDR.address,
-        false
+        true
       );
       expect(disputeResolver.isValid()).is.true;
 
       //Create DisputeResolverFee array so offer creation will succeed
```

Comment:
I know it was pre-existing, but could you also fix this while you're at it? I've noticed these are randomly sprinkled around, we should fix as we go. 
~`//Create`~  `// Create`

Reply:
done